src/der.rs

```diff
@@ -13,7 +13,9 @@
 // OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 use crate::{calendar, time, Error};
-pub(crate) use ring::io::der::{CONSTRUCTED, CONTEXT_SPECIFIC};
+
+pub(crate) const CONSTRUCTED: u8 = 0x20;
```

Comment:
Style nit: top-down ordering would suggest moving these to the bottom of the module.